services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler_mixin.py

```diff
@@ -38,20 +37,20 @@ async def start(self) -> None:
         logger.info("Starting dynamic-sidecar scheduler")
         self._keep_running = True
         self._scheduler_task = asyncio.create_task(
-            self._run_scheduler_task(), name="dynamic-scheduler"
+            self._run_scheduler_task(), name="dynamic-scheduler"  # type: ignore
```

Comment:
why this ignore ?

Reply:
reverted, requires more refactoring in the future. Need to move the methods in the correct place